ods_ci/tests/Resources/Common.robot

```diff
@@ -496,3 +496,12 @@ Clear Element And Input Text
     Clear Element Text    ${element_xpath}
     Sleep    0.5s
     Input Text    ${element_xpath}    ${new_text}
+
+Wait For Dashboard Page Title
```

Comment:
we have the `Wait for RHODS Dashboard to Load` keyword which I think does the same no?